docs/dev/developer-guides/aa.md

```diff
@@ -195,10 +185,9 @@ await aa.deployed();
 
 ### Limitations of the verification step
 
-::: warning Not implemented yet
-
-The verification rules are not fully enforced right now. Even if your custom account works right now, it might stop working in the future if it does not follow the rules below.
-
+::: warning
+- The verification rules are not yet fully enforced. 
+- Even if your custom account works at the moment, it could stop working in the future if it does not follow the rules below.
```

Comment:
Is this really trye @StanislavBreadless ? Given we have versioning?

Reply:
@githubdoramon the reason is that we do not have "Limitations of the verification step" fully implemented yet. Yes, under the same version as well, because we may need to add some restrictions to enhance DDoS protection, etc